torch/csrc/jit/script/module.cpp

```diff
@@ -33,34 +33,34 @@ static ModulePtr create_module_object(
 }
 
 Module::Module(c10::QualifiedName class_name)
-    : module_value_(create_module_object(
+    : Object(create_object_value(
           std::move(class_name),
           std::make_shared<CompilationUnit>())) {}
 
 Module::Module(
     std::shared_ptr<CompilationUnit> cu,
     const c10::ClassTypePtr& type)
-    : module_value_(c10::ivalue::Object::create(
+    : Object(c10::ivalue::Object::create(
           c10::StrongTypePtr(std::move(cu), type),
           type->numAttributes())) {}
 
 Module::Module(
     c10::QualifiedName class_name,
     std::shared_ptr<CompilationUnit> cu,
     bool shouldMangle)
-    : module_value_(create_module_object(
+    : Object(create_object_value(
           std::move(class_name),
           std::move(cu),
           shouldMangle)) {}
 
-ModulePtr Module::module_object() const {
-  if (!module_value_) {
+ObjectPtr Object::object_value() const {
+  if (!object_value_) {
```

Comment:
The behavior in here is specific to Module and not object -- only Module are used in tests this way, and this will break most uses of object. We probably should go through and find the places where `Module m;` is meant to create a blank module, and change it to an explicit call to create a blank module. That way, this code can just assert that `object_value_` exists.